### ISSUE TITLE
Skinwalkers and Insanity Remastered

### ISSUE THREAD

**Issue by mdsl2020** [BudgetAirpods/InsanityRemastered#32]
Not sure if skinwalkers mod or IR mod is causing this but, recordings of your teammates can be heard on the ship while in orbit. It is annoying but not unplayable.

**Comment by BudgetAirpods** [comment]
I'm not sure what would be causing this.  Did you just finish a round when it started or were you just chilling in the lobby?

Only thing I could think of is the fake player failing to de-spawn, which would cause it to speak every few seconds.

**Comment by mdsl2020** [comment]
Yes, this only happens after playing a round (going to the moon and going back to orbit or the company building). It never happens when just starting a game and chilling in the lobby.

**Comment by HossBigft** [comment]
I think it's because you had enabled radios in ship. Yesterday they were talking even in lobby unless we disabled them.

**Comment by mdsl2020** [comment]
> I think it's because you had enabled radios in ship. Yesterday they were talking even in lobby unless we disabled them.

yes it is that, is there any way to disable radios doing this? its very frequent and more annoying than anything.

**Comment by HossBigft** [comment]
> yes it is that, is there any way to disable radios doing this? its very frequent and more annoying than anything.
I tried to disable auditory hallucinations to no avail and didn't recieve reply from author yet, so I guess there is noway. I found workaround though, if you power off radios, they can't talk.

**Comment by Mi6kbuttface** [comment]
> > yes it is that, is there any way to disable radios doing this? its very frequent and more annoying than anything.
> 
> I tried to disable auditory hallucinations to no avail and didn't recieve reply from author yet, so I guess there is no way. I found workaround though, if you power off radios, they can't talk.

I have also encountered this issue, to the point where there were so many fake player voices at once it was unbearable and unable to figure out if someone was talking or it was an audio playback.
